src/clients/third-party-service-client.ts

```diff
@@ -102,6 +103,13 @@ export const getThirdPartyServiceClient =
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         thirdPartyConfig.prodEnvironment ?? thirdPartyConfig.testEnvironment!;
 
+    eventLog.task.info(`Third party client base url: ${environment.baseUrl}`);
```

Comment:
This code will expect a task in input and give a task in output. It should be used inside a pipe.
From [docs](https://github.com/pagopa/winston-ts#how-to-log) it seems we should use `peek`.